tests/toolkit/res_buf_tests.cc

```diff
@@ -12,20 +13,20 @@ namespace toolkit {
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 // - - - - - - Getters, Setters, and Property changers - - - - - - - - - - - -
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-TEST_F(ResBufTest, set_capacity_ExceptionsEmpty) {
+TEST_F(ProductBufTest, set_capacity_ExceptionsEmpty) {
```

Comment:
So in this file you renamed all of the tests to Product instead of Res? Why? Do we still need those ResBuf tests? And if not, it probably makes sense to rename the file to `product_buf_tests`.

Reply:
@gonuke this is from #1687